src/app.js

```diff
@@ -1 +1,39 @@
+/* eslint-disable no-console */
 'use strict';
+
+const rl = require('readline');
```

Comment:
It is better not to shorten the naming to unreadable names, if there are many imports in the project, it will be easy to get confused (`rl` -> `readline`).